schemas/includes/unknown_entity_type.json

```diff
@@ -52,6 +52,12 @@
     "mailing_address": {
       "$ref": "address.json"
     },
+    "identifiers": {
+      "type": ["array", "null"],
```

Comment:
Usual question around allowing `null` vs. just the field being optional being enough. Given the other fields in this schema do the latter, maybe follow that convention here..?